diff --git a/node_modules/duck-duck-scrape/lib/search/search.js b/node_modules/duck-duck-scrape/lib/search/search.js
index 6580879..6c1f821 100644
--- a/node_modules/duck-duck-scrape/lib/search/search.js
+++ b/node_modules/duck-duck-scrape/lib/search/search.js
@@ -36,8 +36,10 @@ async function search(query, options, needleOptions) {
     else
         options = sanityCheck(options);
     let vqd = options.vqd;
-    if (!vqd)
+    if (!vqd) {
         vqd = await (0, util_1.getVQD)(query, 'web', needleOptions || { headers: util_1.COMMON_HEADERS });
+        await new Promise(function (r) { return setTimeout(r, 5000); });
+    }
     /* istanbul ignore next */
     const queryObject = {
         q: query,
